### ISSUE TITLE
% of queries blocked not consistant between web gui and api

### ISSUE THREAD

**Issue by C4Wiz** [pi-hole/AdminLTE#498]
**In raising this issue, I confirm the following (please check boxes, eg [X] - no spaces) Failure to fill the template will close your issue:**

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [x] The issue I am reporting can be *replicated*
- [x] The issue I'm reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the codebase?:**
5
---
**[FEATURE REQUEST | QUESTION | OTHER]:**

Please [submit your feature request here](https://discourse.pi-hole.net/c/feature-requests), so it is votable by the community.  It's also easier for us to track.

**[BUG | ISSUE] Expected Behaviour:**
webpage and api show the same % of queries blocked

**[BUG | ISSUE] Actual Behaviour:**
webpage shows 7.6% and api shows 7%

**[BUG | ISSUE] Steps to reproduce:**

-
-
-
-

**(Optional) Debug token generated by `pihole -d`:**

`<token>`

_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

**Comment by Mcat12** [comment]
What API are you referring to? (api.php, FTL, etc)

**Comment by C4Wiz** [comment]
Api.php

**Comment by Mcat12** [comment]
The web interface uses the results of `api.php?summary` to display the summary. Perhaps the percentage changed between looking at it on the API and web interface?

**Comment by C4Wiz** [comment]
I refreshed them at the same time the api reported 7% and the web ui showed 7.6%. The api only ever shows whole numbers, IE: 7,8,9%

**Comment by C4Wiz** [comment]
Im using this call 'ads_percentage_today'

**Comment by Mcat12** [comment]
What is the full output of your API summary call? (`/admin/api.php?summary`)
Mine is:
```
{"domains_being_blocked":"105,010","dns_queries_today":"15,130","ads_blocked_today":"1,258","ads_percentage_today":"8.3","unique_domains":"2,040","queries_forwarded":"7,949","queries_cached":"5,923"}
```

**Comment by C4Wiz** [comment]
i found the issue, my bitbar script was calling /admin/api.php and was returning this:

`{"domains_being_blocked":"116,913","dns_queries_today":"23,828","ads_blocked_today":"1,838","ads_percentage_today":"7","unique_domains":"1,933","queries_forwarded":"12,056","queries_cached":"9,934"}`

i added ?summary to the call and the result is now consistant

`{"domains_being_blocked":"116,913","dns_queries_today":"23,828","ads_blocked_today":"1,838","ads_percentage_today":"7.7","unique_domains":"1,933","queries_forwarded":"12,056","queries_cached":"9,934"}`

**Comment by C4Wiz** [comment]
Before adding ?summary

![screen shot 2017-05-09 at 12 53 29 pm](https://cloud.githubusercontent.com/assets/3883521/25867366/96c96520-34b6-11e7-8b07-83dbb3faef37.png)


after adding ?summary

![screen shot 2017-05-09 at 12 52 33 pm](https://cloud.githubusercontent.com/assets/3883521/25867392/a94d05c6-34b6-11e7-888c-465e341977e8.png)

**Comment by C4Wiz** [comment]
seems adding ?summary also added "," in the number results

**Comment by DL6ER** [comment]
Was marked "Fixed in Next Release" in May ago. We released Pi-hole v3.1 min-June, so I assume it is fixed by now.
